src/MenuItem/SplitItem.php

```diff
@@ -325,4 +329,23 @@ public function getText() : string
     {
         throw new \BadMethodCallException(sprintf('Not supported on: %s', __CLASS__));
     }
+
+    /**
+     * Finds largest itemExtra value in items
+     */
+    private function calculateItemExtra() : void
```

Comment:
Is this a change in behaviour or just a refactor?

Reply:
> The only thing noteworthy in this particular PR is the addition of SplitItem::calculateItemExtra().
> Since SplitItem does not contain its own styling, and markers/item extra are item-level now vs living in MenuStyle, this method loops through all items assigned to the split and finds the item with the largest item-extra value to calculate column sizes.

Reply:
Yes but what I mean is why. Is that new behaviour, for all columns to be equal size? I don't remember the current behaviour.

Reply:
Checking latest stable tag with this:

```
<?php

use PhpSchool\CliMenu\Builder\SplitItemBuilder;
use PhpSchool\CliMenu\CliMenu;
use PhpSchool\CliMenu\Builder\CliMenuBuilder;

require_once(__DIR__ . '/../vendor/autoload.php');

$itemCallable = function (CliMenu $menu) {
    echo $menu->getSelectedItem()->getText();
};

$builder = (new CliMenuBuilder)
    ->setTitle('Basic CLI Menu Custom Item Extra')
    ->addSplitItem(function (SplitItemBuilder $b) use ($itemCallable) {
        $b->addItem('First Item', $itemCallable, true)
            ->addItem('Second Item', $itemCallable, true)
            ->addItem('Third Item', $itemCallable, true);
    })
    ->setItemExtra('[COMPLETE!]')
    ->addLineBreak('-');

$builder->getStyle()
    ->setDisplaysExtra(true);

$menu = $builder->build();

$menu->open();
```

It seems the item extra string is not displayed when in a split item:

![image](https://user-images.githubusercontent.com/589398/71278528-7b6d3600-231d-11ea-82e9-5241c02bbb10.png)

Should I remove?

This is how it looks in this PR:

![Peek 2019-12-20 11-41](https://user-images.githubusercontent.com/589398/71278614-b2434c00-231d-11ea-8600-42d24266ab3a.gif)

Reply:
Ah yeah, no that's fine :)